core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import "./chips.css";
+
+const Chips = ({ chipsContent, activeChip, onChipClick, theme = "light" }) => {
```

Comment:
Both with `onChipClick` and using `title` as `key` below, you are making an assumption that titles are _unique_, but I think if they are not, we will just get undefined behaviour and a few weird React warnings.

Maybe worth a comment for anyone hunting down weird bugs with this later, but I think it's fine as it is, just wanted to point it out.

Reply:
Yes thought about that too but didn't have a better solution at hand right now. Good to leave a comment though